modules/eslint-plugin/package.json

```diff
@@ -46,7 +46,7 @@
   },
   "peerDependencies": {
     "eslint": "^8.57.0 || ^9.0.0",
-    "@typescript-eslint/utils": "^7.11.0 || ^8.0.0-alpha.20",
+    "@typescript-eslint/utils": "^7.11.0 || ^8.0.0",
```

Comment:
I think the angular eslint plugin should also be upgraded to resolve the issue.
The `alpha` version was added because it was also a peer dependency within the new flat config version of Angular ESLint.

It seems like they upgraded a couple of weeks ago, https://github.com/angular-eslint/angular-eslint/commit/dfd1bd7720cd8ec1524ebd09733c4b5f28f8271e